packages/cli/src/models/files/ProjectFile.ts

```diff
@@ -84,7 +84,8 @@ export default class ProjectFile {
   }
 
   public get root(): string {
-    return path.dirname(this.filePath);
+    // We go to the parent of the containing directory (.openzeppelin) to get the root
+    return path.dirname(path.dirname(this.filePath));
```

Comment:
I don't think this is correct as long as we support the old `zos.json`. Or is migration mandatory?

Reply:
Good catch! Just force pushed a fix. By the way, only after I pushed it I noted I shouldn't have amended, so you could see the new diff. Apologies for that! Still, being a small change, I hope you won't mind!